schema_utils.py

```diff
@@ -229,47 +261,113 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: list(str). An object.
+            min_value: int. The minimum value to check the length of object.
```

Comment:
I don't know what this means. It would be clearer to write something like:

    The minimum number of elements that `obj` should contain.

Ditto below. Make sure your docstrings are clear and readable by others.

Reply:
Done